TWLight/tests.py

```diff
@@ -33,6 +33,8 @@
     ToURequired,
     EmailRequired,
 )
+import pytz
```

Comment:
while this works, it's not very idiomatic for Django, which has it's own timezone handling wrappers.
You should use its tooling instead of `pytz`
https://docs.djangoproject.com/en/3.0/topics/i18n/timezones/